src/data/python-quiz.ts

```diff
@@ -1,4 +1,15 @@
 const pythonQuiz = [
+     {
+    Question: 
+      "In Python, if we have list=['rahul', 'monica', 'george', 'ross', 'george', 'ross'], then what is the result here: print(len(set(list)-len(list))?",
```

Comment:
It looks like there is a syntax error in your code example

```suggestion
      "In Python, if we have list=['rahul', 'monica', 'george', 'ross', 'george', 'ross'], then what is the result here: print(len(set(list))-len(list))?",
```